manifests/inet_filter.pp

```diff
@@ -9,8 +9,8 @@
 
   nftables::config {
     'inet-filter':
-      prefix => '',
-      source => 'puppet:///modules/nftables/config/puppet-inet-filter.nft';
+      prefix  => '',
+      content => "  include \"inet-filter-chain-*.nft\"\n";
```

Comment:
From https://github.com/voxpupuli/puppet-nftables/blob/4e9b7fa3797a939986b50376e867a8039c5bf953/manifests/config.pp#L52C12-L52C12 it looks like this is the default.

```suggestion
```

Reply:
good catch!

Reply:
Except the newline at the end. Is that a problem? I just came across this while reviewing this module for potential usage and found this. Haven't even tried it yet.

Reply:
Looks like that newline doesn't matter here.